README.md

```diff
@@ -1,38 +1,75 @@
-![Threlte Logo](./assets/threlte/logo/threlte_logo_github.png)
+<div align="right">
+ <img alt="license" src="https://img.shields.io/npm/l/@threlte/core?color=fe4100&labelColor=171d27&logo=git&logoColor=white"/>
+ <img alt="npm" src="https://img.shields.io/npm/v/@threlte/core?color=fe4100&labelColor=171d27&logo=npm&logoColor=white"/>
+ <img alt="discord" src="https://img.shields.io/discord/985983540804091964?label=discord&color=fe4100&labelColor=171d27&logo=discord&logoColor=white"/>
+ <img alt="docs" src="https://img.shields.io/website?down_color=red&down_message=offline&label=docs&color=fe4100&labelColor=171d27&up_message=online&url=https%3A%2F%2Fthrelte.xyz&logo=svelte&logoColor=white"/>
+</div>
 
-A three.js component library for svelte.
+<a href="https://threlte.xyz">
+ <img src="./assets/threlte/banner/threlte-banner.png"/>
+</a>
 
-![npm-license](https://img.shields.io/npm/l/@threlte/core?style=flat-square)
-![npm-version](https://img.shields.io/npm/v/@threlte/core?style=flat-square)
-![discord](https://img.shields.io/discord/985983540804091964?label=discord&style=flat-square)
-![website](https://img.shields.io/website?down_color=red&down_message=offline&label=threlte.xyz&style=flat-square&up_message=online&url=https%3A%2F%2Fthrelte.xyz)
+## Create 3D Apps for the Web
 
-Build three.js scenes declaratively with Threlte components that are reactive to the core, readily interactive and can participate in Svelte's ecosystem.
+Threlte is a [Svelte](https://svelte.dev/) library that simplifies creating 3D apps for the web. It provides a **declarative**, **type-safe**, **reactive** and **interactive** API out-of-the-box.
 
-## Official Documentation
+Threlte's core **3D rendering** is powered by [Three.js](https://threejs.org/), but its also provides a **physics engine** through [Rapier](https://rapier.rs/) and an **animation engine** via [Theatre.js](https://www.theatrejs.com/). 
```

Comment:
Suggestion:

"Threlte's **3D rendering** is powered by [Three.js](https://threejs.org/), it provides a **physics engine** through [Rapier](https://rapier.rs/) and an **animation engine** via [Theatre.js](https://www.theatrejs.com/)."